### PR TITLE
Updating lie upstream and move to EAPI 5 issue #415

### PR DESCRIPTION
**Pull request by kiwifb**
Package-Manager: portage-2.2.18

### CODE REVIEW

File: sci-mathematics/lie/lie-2.2.2.ebuild [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-EAPI="3"
+EAPI=5
 
 inherit eutils toolchain-funcs
 
 DESCRIPTION="A Computer algebra package for Lie group computations"
-HOMEPAGE="http://young.sp2mi.univ-poitiers.fr/~marc/LiE"
-SRC_URI="http://young.sp2mi.univ-poitiers.fr/~marc/LiE/conLiE.tar.gz"
+HOMEPAGE="http://www-math.univ-poitiers.fr/~maavl/LiE/"
+SRC_URI="http://wwwmathlabo.univ-poitiers.fr/~maavl/LiE/conLiE.tar.gz"
```
Comment (jlec) [inline sci-mathematics/lie/lie-2.2.2.ebuild]: Could you please add a "-> ${P}.tar.gz" here?

File: sci-mathematics/lie/lie-2.2.2.ebuild [hunk 2]
```diff
@@ -19,24 +19,24 @@ SLOT="0"
...
+RDEPEND="sys-libs/readline:=
 	sys-libs/ncurses"
 
 S="${WORKDIR}/LiE"
 
 src_prepare() {
-	epatch "${FILESDIR}/${P}-make.patch"
-	epatch "${FILESDIR}/parrallelmake-${P}.patch"
+	epatch "${FILESDIR}"/${P}-make.patch
+	epatch "${FILESDIR}"/parrallelmake-${P}.patch
```
Comment (jlec) [inline sci-mathematics/lie/lie-2.2.2.ebuild]: One epatch call is enough.

**Comment by kiwifb** [comment]
Done as requested. Will we able to get just `PATCHES` everywhere in the next EAPI? That'd be neat.

**Comment by jlec** [comment]
Not sure, but you can always do

    PATCHES=( ... )
    src_prepare() {
        epatch "${PATCHES[@]}"
    }

**Pull request merged**